src/api/user-api.js

```diff
@@ -0,0 +1,19 @@
+import config from '~/config';
+import $ from 'jquery';
+
+export function login(username, password) {
+    const payload = {
+        email: username,
+        password,
```

Comment:
Note to self: should hash the password before sending it over the wire.

Don't worry about changing this, as I can handle this with my software security group.